src/azure/cli/_output.py

```diff
@@ -78,6 +54,20 @@ def format_text(obj):
     except TypeError:
         return ''
 
+def format_table(obj):
+    result = obj.result
+    try:
+        if obj.simple_output_query and not obj.is_query_active:
+            if callable(obj.simple_output_query):
```

Comment:
Yep agreed but since I plan to remove the jmespath queries in another PR, I still needed the if statement.
The name will also change from simple_output_query.